### DISCUSSION TITLE
Possible to show axes labels at specific X-values outside of standard interval?

### DISCUSSION THREAD

**Discussion by Tyler-Lopez** [patrykandpatrick/vico#888] [Questions] [answered]
### Question

Hi there, 

Is it currently possible to draw axes labels, possibly including the guideline, at specific X-values outside the standard interval provided? This seems to be a common use-case that I'm not sure is possible in Vico.

For example, instead of just: `0, 25, 50, 75, 100` drawing labels, I might want to throw in an `80`.

Thanks!

### Vico version(s)

2.0.0-alpha.27

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello, @Tyler-Lopez. Yes, this is possible. The default selection of label values is produced by `HorizontalAxis.ItemPlacer.default`. This function has parameters for spacing (which also depends on the value returned by `getXStep`) and offset.

To achieve the result you're looking for, you'll need a simple custom implementation. You can delegate to `HorizontalAxis.ItemPlacer.default()` and just override `getLabelValues` so that it adds 80 to the list returned by `super.getLabelValues`.

**Reply by Tyler-Lopez** [reply]
Awesome!!! Thank you so much for the answer, I will check this out.
